src/quad_expr.jl

```diff
@@ -112,15 +112,31 @@ if VERSION < v"0.7-"
     Base.done(qti::QuadTermIterator, state::Int) = done(qti.quad.terms, state)
     Base.next(qti::QuadTermIterator, state::Int) = reorder_iterator(next(qti.quad.terms, state)...)
 else
-    function reorder_iterator(t::Tuple{Pair{<:UnorderedPair,<:Any},Int})
-        ((first(t).second, first(t).first.a, first(t).first.b), last(t))
+    function reorder_and_flatten(p::Pair{<:UnorderedPair,<:Any})
```

Comment:
`Pair{<:UnorderedPair,<:Any}` is equivalent to `Pair{<:UnorderedPair}` which is arguably simpler/more readable